sync/class.jetpack-sync-module-attachments.php

```diff
@@ -23,6 +23,7 @@ function send_attachment_info( $attachment_id ) {
 		 * @param int The attachment ID
 		 * @param object The attachment
 		 */
-		do_action( 'jetpack_sync_save_add_attachment', $attachment_id, $attachment );
+		$filter = current_filter();
+		do_action( 'jetpack_sync_save_add_attachment', $attachment_id, $attachment, $filter );
```

Comment:
Could you move the action one line up? The parser won't find the docblock if there is another line of code above it.